content/_data/changelogs/weekly.yml

```diff
@@ -10552,6 +10552,80 @@
   # pull: 5356 (PR title: Fix sort order)
   # pull: 5357 (PR title: Test cleanup)
 
+- version: '2.285'
+  date: 2021-03-23
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 4623
+    authors:
+    - daniel-beck
+    message: |-
+      Switch to sending POST requests by default for form validation URLs.
+  - type: rfe
+    category: rfe
+    pull: 5366
+    authors:
+    - dependabot[bot]
+    references:
+      - pull: 5366
+      - url: https://www.jenkins.io/security/advisory/2021-03-18/
+        title: Security Advisory 2021-03-18
+    message: |-
+      Bump matrix-auth from 2.6.5 to 2.6.6.
```

Comment:
Shouldn't we use labels that are shown to admins in Jenkins for changelog entries?